packages/migrate/Migration.js

```diff
@@ -109,17 +113,22 @@ class Migration {
       let artifacts = resolver
         .contracts()
         .map(abstraction => abstraction._json);
-      if (this.config.db && this.config.db.enabled && artifacts.length > 0) {
-        const db = connect(this.config.db);
-        const project = await Project.initialize({
+      if (
+        Db &&
+        this.config.db &&
+        this.config.db.enabled &&
+        artifacts.length > 0
```

Comment:
Not anything to hold back this PR, but this could be refactored out.

Reply:
Let's figure out where to put this :)